README.md

```diff
@@ -2,7 +2,8 @@
 
 A rate limiting library for Starlette and FastAPI adapted from [flask-limiter](http://github.com/alisaifee/flask-limiter).
 
-Note: this is alpha quality code still, the API may change, and things may fall apart while you try it.
+This packaage is used in smaller production setups, and seems to behave as expected. There might be some API changes
```

Comment:
there is a minor typo here, `packaage` > `package`

Reply:
also, maybe we could add some details on what is a "smaller production setup" according to us ?
(btw, we deployed a fastAPI application in production last week, using slowapi and handling millions of requests per month, maybe it can be considered something more than "smaller production setup" ? (even though it's still not the biggest production setup possible :smile:)

Reply:
Thanks for the feedback. This is great news! congratulations on the deployment! I've updated the README based on your comment.